src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -120,7 +139,7 @@ public DrivetrainSubsystem() {
 	 * 'forwards' direction.
 	 */
 	public void zeroGyroscope() {
-		pigeon.setYaw(0.0);
+		pigeon.setYaw(180);
```

Comment:
I am not sure how this will affect autos or other functionality of the robot, so maybe put a marker there so you don't forget about the change later on